.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,36 @@
+name: PythonPackage
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    name: Build and Deploy to PyPi
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # build/push in lowest support python version
+        python-version: [ 3.8 ]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Setup and Build
+      run: |
+        pip install -U pip
+        pip install .[setup]
+        python setup.py sdist bdist_wheel
+        python -m twine check dist/*
+
+    - name: Build and publish
+      uses: pypa/gh-action-pypi-publish@v1.4.2
```

Comment:
I don't think we need a specific version, 1.5.0 is the latest.  `@release/v1` will keep us on 'latest' to some degree.